content/600-about/200-prisma-docs/20-style-guide/03-spelling-punctuation-formatting.mdx

```diff
@@ -239,6 +239,11 @@ const foo = (...) // Too vague
 
 Ensure that code snippets you include are realistic examples that would work if run in the context presented.
 
+### Prisma schema naming conventions
+
+When you create a Prisma schema for an example, follow the [naming conventions](/reference/api-reference/prisma-schema-reference#naming-conventions) we advise for users.
+https://www.prisma.io/docs
```

Comment:
This looks leftover, so remove:
```suggestion
```

Reply:
Thanks - I think I've fixed this in a subsequent push, because I can't see it in my branch now. I'll close this and then double-check the files in the repo

Reply:
Weird - the link was hidden in VSCode. I deleted the line breaks and full stop from the preceding sentence, then put them back in, and GitHub Desktop detected that I'd removed the link! It's gone now, but I'd like to find out what actually happened here.